conjureplugin/conjureplugin.go

```diff
@@ -51,23 +51,54 @@ func Run(params ConjureProjectParams, verify bool, projectDir string, stdout io.
 			GenerateCLI:          currParam.CLI,
 			GenerateFuncsVisitor: currParam.AcceptFuncs,
 		}
+
+		files, err := conjure.GenerateOutputFiles(conjureDef, outputConf)
+		if err != nil {
+			return errors.Wrap(err, "failed to generate conjure output files")
+		}
+
+		var filesToDelete []string
+		if !currParam.SkipDeleteGeneratedFiles {
+			filesToDelete, err = computeObsoleteFiles(outputConf.OutputDir, files)
+			if err != nil {
+				return err
+			}
+		}
+
 		if verify {
-			diff, err := diffOnDisk(conjureDef, projectDir, outputConf)
+			diff, err := diffOnDisk(projectDir, files)
 			if err != nil {
 				return err
 			}
+
+			var msg string
 			if len(diff.Diffs) > 0 {
-				verifyFailedFn(k, diff.String())
+				msg = diff.String()
+			}
+
+			if len(filesToDelete) > 0 {
+				if msg != "" {
+					msg += "\n\n"
+				}
+				msg += "The following generated files will be deleted:\n"
+				for _, file := range filesToDelete {
+					msg += fmt.Sprintf("  - %s\n", file)
+				}
+			}
```

Comment:
Per comment I left in overall PR conversation, this output doesn't match the format/flow of the existing output at all -- after looking at that more closely, I think the better approach here would be to update `diffOnDisk` so that the diff it produces properly accounts for the files that would be delete. Once that is done, all this extra formatting code disappears.